spec/index.bs

```diff
@@ -11550,6 +11589,37 @@ It must only be included by interfaces which also include those mixins.
     Otherwise return `true`.
 </div>
 
+<div algorithm data-timeline=device>
+    <dfn abstract-op>Validate immediate data</dfn>(encoder, pipeline)
```

Comment:
Since we are validating immediates at draw-time. What's the use of `GPUPipelineDescriptor.immediateSize`.

Reply:
Resolved in #5473